assessments/projects/nats/self-assessment.md

```diff
@@ -278,7 +278,7 @@ Security issues can be reported through various channels:
 Additionally, 
 
 
-### [CII Best Practices]([https://www.coreinfrastructure.org/programs/best-practices-program/](https://www.bestpractices.dev/en/projects/1895))
+### [CII Best Practices](https://www.coreinfrastructure.org/programs/best-practices-program/)
```

Comment:
The build fails without this change now that all assessments are being brought in as sub-pages